blueprint/security/hybrid-system-security-plan.md

```diff
@@ -821,20 +841,28 @@ The HybridSystem leverages Azure AD for controlling system access. All technical
 
 The HybridSystem utilises RBAC, automation and policy controls to restrict access to modify any of the functional capabilities provided by the HybridSystem. The HybridSystem also provides auditing and alerting on attempted or successful modifications of the HybridSystem capabilities. 
 
-The HybridSystem provides security controls and an identity management framework that can be utilised to manage system access for systems deployed within the HybridSystem. The HybridSystem enforces multi-factor authentication through conditional access policies, creates recovery accounts for maintaining access to resources and enforces password policies for accounts created directly in Azure AD. The HybridSystem uses a soft-token to reduce the need for purchase, distribution and management of hard-tokens.
+The HybridSystem provides security controls and an identity management framework that can be utilised to manage system access for systems deployed within the HybridSystem. The HybridSystem enforces multi-factor authentication through Conditional Access policies, creates recovery (emergency access) accounts for maintaining access to resources and enforces password policies for accounts created directly in Azure AD. The HybridSystem uses a soft-token - the Microsoft Authenticator app - to reduce the need for purchase, distribution and management of hard-tokens.
+
+The Microsoft Authenticator app cannot be centrally managed to require the enforcement of a PIN or biometric unlock, and is not considered verifier impersonation resistant.
+
+The Protected User security group is not available in Azure AD and is therefore not used by the HybridSystem.
```

Comment:
suggest adding that it is an agency responsibility to configure the protected users group for on-prem AD instance that is sync'd to Azure AD, as this reads that it doesn't exist for hybird.

Reply:
Agreed. Updated compliance approach and added to residual controls for Agency.